app/code/community/Ebizmarts/MailChimp/Helper/Data.php

```diff
@@ -4918,4 +4908,33 @@ protected function getDateHelper()
     {
         return Mage::helper('mailchimp/date');
     }
+
+    /**
+     * @param $scopeId
+     * @param $scope
+     * @param $where
+     * @throws Mage_Core_Exception
+     */
+    protected function removeMessageErrorsFromNewsletterSubscriber($scopeId, $scope, $where)
```

Comment:
Rename to removeErrorsFromSubscribers to maintain the shortest name possible without loosing relevant information.

Reply:
Solved